layout/_partial/sidebar.ejs

```diff
@@ -27,9 +27,9 @@
             <% for (var n in theme.sidebar[i]) { %>
                 <li class="sidebar-button">
                     <% if (url_for(theme.sidebar[i][n].url).indexOf(config.url) < 0 && url_for(theme.sidebar[i][n].url).indexOf(':') >= 0) { %>
-                        <a  class="sidebar-button-link <% if (theme.sidebar[i][n].class) { %><%= theme.sidebar[i][n].class %><% } %>" href="<%- url_for(theme.sidebar[i][n].url) %>" target="_blank" rel="noopener">
+                        <a itemprop="sameAs" class="sidebar-button-link <% if (theme.sidebar[i][n].class) { %><%= theme.sidebar[i][n].class %><% } %>" href="<%- url_for(theme.sidebar[i][n].url) %>" target="_blank">
```

Comment:
This happened because I did edit files on my blog for test purposes and, there, I am using v1.9.1 so, differences between there and dev branch are expected. However, I did not pay attention that, if I cherry pick files from there to my theme fork, git would "subtract" differences (which make sense a lot).